FreeRTOS-Plus/Demo/AWS/Device_Defender_Windows_Simulator/Device_Defender_Demo/DemoTasks/DefenderDemoExample.c

```diff
@@ -19,13 +19,33 @@
  * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  *
- * http://www.FreeRTOS.org
- * http://aws.amazon.com/freertos
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
  *
+ * 1 tab == 4 spaces!
+ */
+
+/*
+ * Demo for showing how to use the Device Defender library's API. This version
+ * of the Device Defender API provides macros and helper functions for
+ * assembling MQTT topics strings, and for determining whether an incoming MQTT
+ * message is related to device defender. The Device Defender library does not
+ * depend on a MQTT library, therefore the code for MQTT connections is placed
+ * in another file (mqtt_demo_helpers.c) to make it easy to read the code using
+ * the Device Defender library.
+ *
+ * This demo connects to the AWS IoT broker and subscribes to the device
+ * defender topics. It then collects metrics for the open ports and sockets on
```

Comment:
I think it will be better to have an introductory description about Device defender service as well. Also please highlight that this demo can only work with AWS IoT broker as Defender is an AWS service.